exp-player/addon/templates/components/exp-video-setup.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="row">
+  <h3> {{ title }} </h3>
```

Comment:
Informational only:

Some devices treat heading levels as having special meaning; try to avoid using non-sequential tags (h3-->h5) purely for formatting purposes. 
http://accessibility.psu.edu/headingshtml/

> Many tags in HTML, including the Heading tags (H1,H2,H3…H6) were developed not to assist formatting but to provide information on the structural hierarchy of a document... Many experts recommend reserving H1 for the page title, H2 for major headings and H3 for major sub headings.